include/groonga/groonga.h

```diff
@@ -1280,6 +1280,20 @@ grn_obj_get_range(grn_ctx *ctx, grn_obj *obj);
 #define GRN_OBJ_GET_DOMAIN(obj)                                                \
   ((obj)->header.type == GRN_TABLE_NO_KEY ? GRN_ID_NIL : (obj)->header.domain)
 
+/**
+ * \brief Free allocatable memory occupied by an object based on a threshold.
+ *
+ * \note This function is not implemented yet.
+ *       The \ref grn_obj_expire function is intended to serve as a
+ *       generic version of grn_ii_expire and grn_io_expire.
```

Comment:
Rubyだとメソッド名を`Class#method`と表記するように、Cだと関数には`()`をつける慣習があるんですよ。

```suggestion
 *       generic version of grn_ii_expire() and grn_io_expire().
```